pkg/azure/table/credentials.go

```diff
@@ -0,0 +1,86 @@
+package table
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"get.porter.sh/plugin/azure/pkg/azure/azureconfig"
+	"get.porter.sh/plugin/azure/pkg/azure/common"
+
+	"github.com/Azure/azure-sdk-for-go/profiles/latest/storage/mgmt/storage"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+	"github.com/hashicorp/go-hclog"
+	"github.com/pkg/errors"
+)
+
+const ConnectionEnvironmentVariable = "AZURE_STORAGE_CONNECTION_STRING"
+const UserAgent = "porter.azure.storage.plugin"
+
+func GetCredentials(cfg azureconfig.Config, l hclog.Logger) (string, string, error) {
```

Comment:
Perhaps for a follow-up, but would either/both of `GetCredentials` and `GetCredentialsFromCli` be candidates for inclusion into the newly-formed `common` pkg?  There look to be a few variants of these spread across table and blob storage pkgs...

Reply:
Yeah there are a few things like this that would be good to do, I'll take a look once its merged